Backend/src/modules/account/tests/create_transfer.rs

```diff
@@ -0,0 +1,617 @@
+use rocket::local::Client;
+use rocket::http::{ContentType, Status};
+use crate::modules::account::dto::{CreateMember, Credentials};
+use crate::modules::account::Account;
+use mysql_connection::tools::{Exists, Execute};
+
+#[test]
+fn create_account_nick_valid_email_valid_password_valid() {
+    // Given
+    let account = Account::default();
```

Comment:
There is a lot of duplicated code. You could also use private functions to shorten it. I will look into a JUnit like syntax crate though soon.

Reply:
> You could also use private functions to shorten it.
Will do (but not before the weekend 🤯 )